### ISSUE TITLE
largeop="true" does not render intersection symbol correctly

### ISSUE THREAD

**Issue by svejdani** [mathjax/MathJax#2912]
When rendering intersection symbol with largeop="true", the symbols are both rendered the same size with only a shift in y-axis.

MathML: `<math xmlns="http://www.w3.org/1998/Math/MathML"><mo>&#x222A;</mo><mo largeop="true">&#x222A;</mo></math>` 

Expected result: 
![image](https://user-images.githubusercontent.com/105304186/183519339-78f9ada2-ce79-40f3-91f2-e1c556e620b0.png)

MathJax Render: 
![image](https://user-images.githubusercontent.com/105304186/183519399-1b3ce64f-cd63-4a2a-aac6-4c8e376580e0.png)

Technical details:
MathJax Version: Latest (Using demo page) https://mathjax.github.io/MathJax-demos-web/input-mml2svg.html
Client OS: Mac
Browser: Chrome Version 103.0.5060.114 (Official Build) (64-bit)
Renderer: MathML to SVG

**Comment by dpvc** [comment]
If you want a large operator (that corresponds to `\bigcup` in TeX), then you want to use U+22C3 not U+222A.  The U+222A character is the binary union operator, while U+22C3 is the *n*-ary large operator.  MathJax only has one size of U+222A, but has small and large versions of U+22C3.  The vertical shift is because `largeop=true` also causes the operator to be centered on the math axis, while the original character isn't quite centered.
